pkg/providers/aws/provider_postgressnapshot_test.go

```diff
@@ -83,8 +83,9 @@ func buildRdsClientMock(modifyFn func(*rdsClientMock)) *rdsClientMock {
 func buildTestPostgresSnapshotCr() *v1alpha1.PostgresSnapshot {
 	return &v1alpha1.PostgresSnapshot{
 		ObjectMeta: controllerruntime.ObjectMeta{
-			Name:      "test",
-			Namespace: "test",
+			Name:            "test",
+			Namespace:       "test",
+			ResourceVersion: fakeResourceVersion,
```

Comment:
Just a question. What caused the need to add this ResourceVersion to the object?